apps/storybook/src/stories/internal-ui/Select/Select.stories.tsx

```diff
@@ -0,0 +1,285 @@
+      table: {
+        defaultValue: {
+          summary: 'false',
+        },
+        type: {
+          summary: 'boolean',
+        },
+      },
+    },
+    required: {
+      control: 'boolean',
+      description: 'Select required',
+      table: {
+        defaultValue: {
+          summary: 'false',
+        },
+        type: {
+          summary: 'boolean',
+        },
+      },
+    },
+    feedback: {
+      control: 'text',
+      description: 'Select feedback',
+      table: {
+        type: {
+          summary: 'string',
+        },
+      },
+    },
+    value: {
+      control: 'text',
+      description: 'Select current value',
+      table: {
+        type: {
+          summary: 'string | number | null',
+        },
+      },
+      type: {
+        required: true,
+        name: 'other',
+        value: 'string | number | null',
+      },
+    },
+    displayValue: {
+      control: 'text',
+      description: 'Select current display value',
+      table: {
+        type: {
+          summary: 'string | number | null',
+        },
+      },
+      type: {
+        required: true,
+        name: 'other',
+        value: 'string | number | null',
+      },
+    },
+    onSelect: {
+      action: 'clicked',
+      description: 'Select on select',
+      table: {
+        type: {
+          summary: 'function',
+        },
+      },
+      type: {
+        required: true,
+        name: 'function',
+      },
+    },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Select>;
+
+export const Default = {
+  render: ({ label = 'Select Label', ...args }: SelectProps<string>) => {
+    const [value, setValue] = useState<string>('');
+
+    const onSelect = ({ value }: SingleSelectOnSelectParams<string>) => {
+      setValue(value);
+    };
+
+    return (
+      <Select
+        className='w-[15rem]'
+        label={label}
+        {...args}
+        value={value}
+        onSelect={onSelect}
+        displayValue={value}
+      >
+        {Array.from({ length: 10 }, (_, idx) => (
+          <Select.Item key={idx} value={String(idx)}>
+            {idx}
+          </Select.Item>
+        ))}
+      </Select>
+    );
+  },
+};
+
+export const WithBadge = {
+  render: () => {
+    const [value, setValue] = useState<string>('');
+
+    const onSelect = ({ value }: SingleSelectOnSelectParams<string>) => {
+      setValue(value);
+    };
+
+    return (
+      <Select
+        className='w-[15rem]'
+        label='Select Label'
+        value={value}
+        onSelect={onSelect}
+        displayValue={value}
+        badge={<Icon iconKey='info' weight='fill' />}
+      >
+        {Array.from({ length: 10 }, (_, idx) => (
+          <Select.Item key={idx} value={String(idx)}>
+            {idx}
+          </Select.Item>
+        ))}
+      </Select>
+    );
+  },
+};
+
+export const Error: Story = {
+  args: {
+    isError: true,
+    feedback: '에러가 발생했어요.',
+  },
+  render: ({ isError, feedback }) => {
```

Comment:
p1: 하기 props 빼서 쓰는 Story 들 타입 지정이 빠져서 타입 에러나는 것 같아요! 
```suggestion
  render: ({ isError, feedback }: SelectProps<SelectValue>) => {
```